app/controllers/integrations/base_controller.rb

```diff
@@ -45,10 +45,14 @@ def deploy?
     raise NotImplementedError, "#deploy? must be overridden in a subclass"
   end
 
+  def release_params
```

Comment:
It was actually our original attempt, but turn out this is rather BuildKite specific, cos the build_param method is only accessible in BuildKiteController. So that implies different CI gonna get their number differently which make sense i think :)